Samples/LabServices/DotnetSDK/Instructions.txt

```diff
@@ -0,0 +1,16 @@
+How to set up a simple app to use the .NET SDK for Azure Lab Services
```

Comment:
Make this a `README.md` and it should get picked up as the folder's documentation. Use formatting accordingly.